tests/test_browsing_context.py

```diff
@@ -922,3 +922,37 @@ async def test_browsingContext_ignoreCache(websocket, context_id, ignoreCache):
         "id": id,
         "result": {},
     }
+
+
+@pytest.mark.asyncio
+async def test_browsingContext_fragmentNavigated_event(websocket, context_id,
+                                                       html):
+    url = html()
+
+    await subscribe(websocket, ["browsingContext.fragmentNavigated"])
```

Comment:
Nit: when listening to a single element,  you can pass a string directly (instead of an array).

Reply:
ack